src/tests/test_dynamic_api/test_serializers.py

```diff
@@ -2,9 +2,13 @@
 
 import pytest
 
+from gisserver.types import CRS
 from dso_api.dynamic_api.serializers import serializer_factory
 from rest_framework_dso.fields import EmbeddedField
 
+RD_NEW = CRS.from_string("EPSG:28992")  # Amersfoort / RD New
+ETRS89 = CRS.from_string("EPSG:4258")  # European Terrestrial Reference System 1989
```

Comment:
misschien moeten we deze maar eens centraal gaan neerzetten, nu we zo zo vaak aanmaken.
`dso_api.lib.spatial`?